iOS/Flows/Conversation/Threads/ConversationThreadViewController.swift

```diff
@@ -135,6 +135,10 @@ class ConversationThreadViewController: DiffableCollectionViewController<Convers
         
         return data
     }
+
+    override func getAnimationCycle() -> AnimationCycle? {
+        return nil
+    }
```

Comment:
Returning nil means the data will be loaded without an animation cycle. (It just fades in).